frontend/components/forms/RegistrationForm/OrgDetails/OrgDetails.jsx

```diff
@@ -49,11 +49,11 @@ class OrgDetails extends Component {
             {...fields.org_logo_url}
             placeholder="Organization Logo URL (optional)"
             tabIndex={tabIndex}
-            hint="must start with https://"
+            hint="Set a logo to use in the Fleet application. Should be an https URL to an image file (e.g. https://fleetdm.com/logo.png)."
```

Comment:
This is definitely more helpful. Do you know the dimensions we expect for this?

Could we easily make https://fleetdm.com/logo.png actually serve an asset that would fit will there (in case anyone looks at it as an example)?

Reply:
@noahtalerman @edamamedesign how about we specify the ideal px dimensions for the image in the help text?  Even though there is wiggle room with the exact pixels, some guidance helps

@zwass good call

Reply:
https://github.com/fleetdm/fleetdm.com/pull/10

Reply:
https://github.com/noahtalerman/fleet-1/commit/aaee5d257bdc270583799d65de7be0504a1ae484 tweaks the verbiage to add a recommendation about dimensions+aspect ratio

Reply:
Great call Zach. The verbiage change looks good. Thanks Mike!